src/app/tables/artifacts-table/artifacts-table.component.spec.ts

```diff
@@ -82,7 +82,7 @@ describe('ArtifactsTableComponent', () => {
     // Make sure the correct number of columns show as selected and that only 20 columns
     // are available since "Resource Type" and "Display Name" should be excluded
     const activeColumnsText = fixture.debugElement.query(By.css('.estuary-table-header__columns-text')).nativeElement;
-    expect(activeColumnsText.textContent.trim()).toBe('6 of 20 columns selected');
+    expect(activeColumnsText.textContent.trim()).toBe('6 of 21 columns selected');
```

Comment:
The `6` should be `7`. Perhaps you can subtract 1 from `this.numColumns` if `this.sideColumnName` is set.

Reply:
Would it be better to have it as 6 of 20 since the additional column is not a normal one, or does 7 of 21 make more sense?